plugins/org.yakindu.sct.doc.user/src/user-guide/editing_statecharts.textile

```diff
@@ -511,3 +511,125 @@ h3(#oss_contributing_examples). Contributing own examples
+
+h4. Showing transition priorities
+
+In order to display "transition priorities":statechart_language.html#transition-priorities in a statechart, set a check mark at the respective preference option. Section "&quot;Transition priorities&quot;":statechart_language.html#transition-priorities has an example.
+
+h4. Disabling live validation
+
+While you are editing a statechart, the statechart editor validates your model on each and every modification you make. That's very helpful, because the editor provides you with instant feedback, so you can see immediately what is right, wrong, or dubious, and you can take corrective action, if needed.
+
+However, on very large and complex statechart models, validation may take a considerable amount of time, causing delays and impeding your editing. Remove the check mark from _Enable live validation_, and your model will be validated before it is saved to the model file only.
+
+h3. Example wizard
+
+p(#fig_preferences-example-wizard). 
+!(standard-image)images/docu_preferences_020_examples_01.png(Preferences: Example wizard)!
+
+p=. Preferences: Example wizard
+
+You can change the example wizard storage location here. It is a local directory where the "example wizard":#oss_example_wizard stores your local clone of the examples Git repository. Default is the _sct_examples_ directory in your home directory.
+
+h3. Expressions
+
+p(#fig_preferences-expressions). 
+!(standard-image)images/docu_preferences_030_expressions_01.png(Preferences: Expressions)!
+
+p=. Preferences: Expressions
+
+Certain dialogs in the statechart editor allow you to opt for never seeing them again. By clicking on the _Clear_ button on this preference pane these hidden dialogs will be shown again at their respective locations.
+
+h4. Expression syntax coloring
+
+p(#fig_preferences-expression-syntax-coloring). 
+!(standard-image)images/docu_preferences_040_expressions_syntax_coloring_01.png(Preferences: Expression syntax coloring)!
+
+p=. Preferences: Expression syntax coloring
+
+This preferences pane defines foreground color, background color, font, and style for displaying certain syntactical elements in statechart language texts.
+
+h4. Expression templates
+
+p(#fig_preferences-expression-templates). 
+!(standard-image)images/docu_preferences_050_expressions_template_01.png(Preferences: Expression templates)!
+
+p=. Preferences: Expression templates
+
+Templates are sections of textual code that occur frequently enough that you would like to be able to insert them with a few keystrokes. This function is known as _content assist_; the sections of code that are inserted are known as _templates_.
+
+To insert an existing content assist template, type the initial character, then press @[Ctrl+Space]@. The templates that begin with that character appear. Double-click on a template to insert it.
+
```

Comment:
Actually, I think the present description is correct: "On this preference pane you can create, edit, and delete **statechart language templates**", i.e. it applies to the whole SText language, not only to expressions. One might argue, however, that the title "Expressions" is misleading and should be changed.